src/core/hle/kernel/process.h

```diff
@@ -216,6 +216,16 @@ class Process final : public SynchronizationObject {
         total_process_running_time_ticks += ticks;
     }
 
+    /// Gets the process schedule count, used for thread yelding
+    constexpr s64 GetScheduledCount() const {
```

Comment:
```suggestion
    s64 GetScheduledCount() const {
```

Process will never be constructed in a constexpr context. Ditto for `IncrementScheduledCount`